### ISSUE TITLE
reflow and components not modified by the author

### ISSUE THREAD

**Issue by bruce-usab** [w3c/wcag#4722]
It seems inconsistent that Reflow SC 1.4.10 doesn’t include an exception for UI components which are not modified by the author.

SC with the exception:

[SC 1.4.11 Non-text Contrast](https://www.w3.org/TR/WCAG22/#non-text-contrast) 
> …except for inactive components or where the appearance of the component is determined by the user agent and not modified by the author;

[SC 1.4.13 Content on Hover or Focus](https://www.w3.org/TR/WCAG22/#content-on-hover-or-focus)
> Exception: The visual presentation of the additional content is controlled by the user agent and is not modified by the author.

[SC 2.4.13 Focus Appearance](https://www.w3.org/TR/WCAG22/#focus-appearance)
> - The focus indicator is determined by the user agent and cannot be adjusted by the author, or
> - The focus indicator and the indicator‘s background color are not modified by the author.

[SC 2.5.5 Target Size (Enhanced)](https://www.w3.org/TR/WCAG22/#target-size-enhanced) and [SC 2.5.8 Target Size (minimum)](https://www.w3.org/TR/WCAG22/#target-size-minimum)
> **User Agent Control:** The size of the target is determined by the user agent and is not modified by the author;

[SC 2.5.7 Dragging Movements](https://www.w3.org/TR/WCAG22/#dragging-movements)
> …unless dragging is essential or the functionality is determined by the user agent and not modified by the author.

This issues was prompted by discussion on #1233 on Backlog call 31 October 2025.

**Comment by bruce-usab** [comment]
Should _all_ of these be of the form “_[something]_ determined by the user agent and not modified by the author”?  When writing standards, it’s foundational to use consistent phrasing.
- With 2.4.13, “cannot be adjusted by the author” seems overly strict, and perhaps untestable.
- With 1.4.13, is there a difference between “controlled by” versus “determined by”?  If so, what is an example?

In addition to 1.4.10 Reflow, I want to check if other SC warrant this exception.
